apps/showcase/src/app/app.fixture.ts

```diff
@@ -0,0 +1,58 @@
+import { ComponentFixtureProfile, O3rComponentFixture } from '@o3r/testing/core';
+import { SidenavPresFixture, SidenavPresFixtureComponent } from '../components/utilities/sidenav/sidenav-pres.fixture';
+
+/**
+ * A component fixture abstracts all the interaction you can have with the component's DOM
+ * for testing purpose, including instantiating the fixtures of sub-components.
+ * It should be used both for component testing and automated testing.
+ */
+export interface AppFixture extends ComponentFixtureProfile {
+  getSideNav(): Promise<SidenavPresFixture>;
+  navigateToHome(): Promise<void>;
+  navigateToRunAppLocally(): Promise<void>;
+  navigateToConfiguration(): Promise<void>;
+  navigateToLocalization(): Promise<void>;
+  navigateToDynamicContent(): Promise<void>;
+  navigateToRulesEngine(): Promise<void>;
+  navigateToComponentReplacement(): Promise<void>;
+  navigateToSDKGenerator(): Promise<void>;
+}
+
+export class AppFixtureComponent extends O3rComponentFixture implements AppFixture {
+  public async getSideNav() {
+    return new SidenavPresFixtureComponent(await this.query('o3r-sidenav-pres'));
+  }
+
+  public async navigateToHome() {
```

Comment:
should we add jsdoc for public methods or we don't apply this rule for fixtures?